codec/src/main/java/io/netty/handler/codec/DefaultTextHeaders.java

```diff
@@ -625,4 +487,130 @@ private CharSequence commaSeparateEscapedValues(CharSequence currentValue, CharS
          */
         CharSequence escape(T value);
     }
+
+    private static final class CharSequenceToStringConverter implements TypeConverter<CharSequence, String> {
+
+        private static final CharSequenceToStringConverter INSTANCE = new CharSequenceToStringConverter();
```

Comment:
This has changed in #4038...lets re-review there. `CharSequenceToStringConverter` is a top level class and `DefaultTextHeaders` no longer exists.